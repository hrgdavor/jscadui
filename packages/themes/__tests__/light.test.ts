import * as fs from 'fs'
import path from 'path'

const FILE_PATH = path.resolve(__dirname, '..', 'themes', 'light.ts')

const theme = fs.readFileSync(FILE_PATH, {
  encoding: 'utf8',
})

it('generated index.ts matches snapshot', () => {
  expect(theme).toMatchSnapshot()
})
