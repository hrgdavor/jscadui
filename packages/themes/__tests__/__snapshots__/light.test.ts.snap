// Vitest Snapshot v1

exports[`generated index.ts matches snapshot 1`] = `
"export const light = {
  name: 'Light',
  color: [0, 0.6, 1, 1],
  bg: [1, 1, 1, 1],
  grid1: [0, 0, 0, 0.2],
  grid2: [0, 0, 1, 0.1],
}
"
`;
